tests/fixture/tmpnet/node.go

```diff
@@ -154,6 +155,11 @@ func ReadNodes(networkDir string, includeEphemeral bool) ([]*Node, error) {
 			continue
 		}
 
+		// Ensure the node ID is set
+		if err := node.EnsureNodeID(); err != nil {
```

Comment:
nit comment feels unneeded

```suggestion
		if err := node.EnsureNodeID(); err != nil {
```